service/src/main/java/org/cbioportal/service/GenericAssayService.java

```diff
@@ -15,4 +15,12 @@
     List<GenericAssayData> getGenericAssayDataInMultipleMolecularProfiles(List<String> molecularProfileIds, 
                                                                         List<String> sampleIds, List<String> genericAssayStableIds, String projection)
         throws MolecularProfileNotFoundException;
+
+    List<GenericAssayData> getGenericAssayData(String molecularProfileId, String sampleListId, 
+                                            List<String> genericAssayStableIds, String projection) 
+        throws MolecularProfileNotFoundException;
```

Comment:
Is it not nicer to create an overloaded method `getGenericAssayData` that takes either a single or multitple molecularProfileId(s)?

```
List<GenericAssayData> getGenericAssayData(String molecularProfileId, ...
List<GenericAssayData> getGenericAssayData(List<String> molecularProfileIds, 
```

Reply:
Yeah, I'm not sure what the difference in method names (get vs. fetch) serves here.

Reply:
@n1zea144 I just referenced the [MolecularDataService](https://github.com/cBioPortal/cbioportal/blob/master/service/src/main/java/org/cbioportal/service/MolecularDataService.java). Do you want me to change to other names?